docs/pages/management/register-services/register-services-auth-service.mdx

```diff
@@ -1,45 +1,68 @@
 ---
-title: Adding Nodes to your Cluster
-description: How to add Nodes to your Teleport cluster
+title: Register Services via the Auth Service
+description: How to register services directly with the Teleport Auth Service via a join token
 ---
 
-This guide explains how to add Teleport Nodes to your cluster.
+This guide will show you how to register a Teleport service with your cluster by
+presenting a **join token** directly to the Teleport Auth Service, without
+connecting via the Proxy Service.
+
+In this approach, you declare your intention to register a new service, and
+Teleport generates a secure token for the service. When the service starts, it
+presents the token, indicating to your Teleport cluster that it is trustworthy.
+
+<Admonition type="warning" title="Before following this guide">
+
+In most cases, you should be able to register Teleport services via the Proxy
+Service using a join token or another method. See our overview of [Registering
+Services](../registering-services.mdx) for all available methods.
+
+Only connect services directly to the Auth Service if no other methods are
+suitable, as we recommend exposing the Auth Service to as few sources of ingress
+traffic as possible.
+
+</Admonition>
 
 ## Prerequisites
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- A Linux server that you will use to host your Teleport Node.
+- A Linux server that you will use to host your Teleport service. 
+
+  In this guide, we will show you how to register a Teleport SSH Service
+  instance. This approach applies to new Proxy Service instances as well as the
+  Kubernetes Service, Database Service, and other Teleport services for
+  accessing resources in your infrastructure.
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/3. Install Teleport on your Node
+## Step 1/3. Install Teleport
 
-On the host where you will run your Teleport Node, follow the instructions for
-your environment to install Teleport.
+Install Teleport on the host where you will run your Teleport service.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-## Step 2/3. Join your Node to the cluster
+## Step 2/3. Join your service to the cluster
```

Comment:
Technically, `instance` would be a more correct term, as multiple services can run in a single Teleport process.